test/functional/ex_cmds/mksession_spec.lua

```diff
@@ -91,7 +91,7 @@ describe(':mksession', function()
     command('tabnext 1')
     eq(cwd_dir .. get_pathsep() .. tmpfile_base .. '1', funcs.expand('%:p'))
     command('tabnext 2')
-    eq(cwd_dir .. get_pathsep() .. tmpfile_base .. '2', funcs.expand('%:p'))
+    eq(cwd_dir .. '/' .. tmpfile_base .. '2', funcs.expand('%:p'))
```

Comment:
This is unexpected. `shellslash` is not set so `expand("%:p")` shouldn't use "/".